0008-security-recommendations/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: KP Security Recommendations
+shortname: 8/Security
+status: raw
+editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
+---
+## Introduction
+This document describes security recommendations for systems using 1/KP. This is only a recommendation and is not mandatory to implement.
```

Comment:
[One sentence per line](http://kaaproject.github.io/kaa/docs/v0.10.0/How-to-contribute/Style-guide/#markdown-formatting) in markdown please.